releases/release-1.31/release-notes/maps/pr-125450-map.yaml

```diff
@@ -0,0 +1,4 @@
+pr: 125450
+releasenote:
+  text: pkg k8s.io/apiserver/pkg/storage/cacher, method (*Cacher) Wait(context.Context)
+    error
```

Comment:
Not sure what this mean. was this error fix? are we catching this error now? Need clarification.

Reply:
@rayandas @shecodesmagic made the changes thanks